front-end/src/components/productView/ProductView.js

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { productService } from '../../services/productServices';
+
+const Container = styled.div`
+    display: flex;
+    justify-content: space-evenly;
+    padding: 2rem 0 2rem 0;
```

Comment:
This can be simplified.
```suggestion
    padding: 2rem 0;
```